app/src/main/res/values/strings.xml

```diff
@@ -480,4 +480,14 @@
   <string name="developer_options_force_crash">Force crash</string>
   <string name="all">All</string>
   <string name="mark_completed_uppercase">MARK COMPLETED</string>
+  <!-- ThirdPartyDependencyListActivity -->
+  <string name="third_party_dependency_list_activity_title">Third-party Dependencies</string>
+  <string name="third_party_dependencies">Third Party Dependencies</string>
+  <string name="third_party_dependency_version">v %s</string>
+
+  <!-- LicenseListActivity -->
+  <string name="license_list_activity_title">Copyright Licenses</string>
```

Comment:
Will this be replaced by the name of the dependency/is this just a placeholder?

Reply:
No, it will not be replaced by the dependency name. The activity will actually display the list of licenses associated with the clicked dependency.